guide/src/building_and_distribution.md

```diff
@@ -118,6 +118,11 @@ cargo build --target x86_64-pc-windows-gnu
 
 Any of the `abi3-py3*` features can be enabled instead of setting `PYO3_CROSS_PYTHON_VERSION` in the above examples.
 
+As an advanced feature, `PYO3_CROSS_NO_ENABLE_SHARED` can be set to enable support for static linking
+of the Python interpreter library when targeting Windows.
+It can only be used for building Rust applications embedding a Python interpreter,
+and is ignored for the Python extension modules (when `extension-module` feature is enabled).
```

Comment:
```suggestion
It is only relevant when building Rust applications embedding a Python interpreter,
and so is ignored for Python extension modules (when the `extension-module` feature is enabled).
```

We should also comment on the downsides here - only symbols used in the program will be included, so importing modules in the embedded program will often fail.

Reply:
On Windows extensions can only import symbols from a Python DLL (either python3y.dll or python3.dll). When the interpreter is built as a static library no interface DLL is created, and dynamically loading extension modules will not work at all. This is different from *nix, where the extensions can grab symbols from the executable.